scripts/run_integration_tests_aarch64.sh

```diff
@@ -102,6 +102,20 @@ build_edk2() {
     popd
 }
 
+# stress is a tool used by test case `test_virtio_balloon`
+build_stress() {
+    STRESS_REPO="https://salsa.debian.org/debian/stress.git"
+    STRESS_DIR="$WORKLOADS_DIR/stress"
+    if [ -d "${STRESS_DIR}" ]; then
+        rm -rf "${STRESS_DIR}"
+    fi
+
+    checkout_repo "$STRESS_DIR" "$STRESS_REPO" debian/master
+    pushd "$STRESS_DIR"
+    ./configure --enable-static && make
+    popd
+}
```

Comment:
Don't we want the image to contain stress directly? I mean instead of building it on every PR?

Reply:
Hi @sboeuf. Of course that would be perfect. However we (me an Michael) do not have access to the `https://cloud-hypervisor.azureedge.net/` so that we cannot modify the cloud image stored there directly. Also, we are not allowed to upload anything but source code to upstream.

Based on these two reasons, this is probably the only method to achieve our goal....

Reply:
Yeh, that makes the most sense to me.